src/main/java/org/sunbird/profile/service/UserBulkUploadService.java

```diff
@@ -69,9 +69,14 @@ public void initiateUserBulkUploadProcess(String inputData) {
             if (errList.isEmpty()) {
                 updateUserBulkUploadStatus(inputDataMap.get(Constants.ROOT_ORG_ID),
                         inputDataMap.get(Constants.IDENTIFIER), Constants.STATUS_IN_PROGRESS_UPPERCASE, 0, 0, 0);
-                storageService.downloadFile(inputDataMap.get(Constants.FILE_NAME));
-                //processBulkUpload(inputDataMap);
-                processCSVBulkUploadV2(inputDataMap);
+                String fileName = inputDataMap.get(Constants.FILE_NAME);
+                logger.info("fileName {} ", fileName);
+                storageService.downloadFile(fileName);
+                if (fileName.endsWith(Constants.CSV_FILE)) {
```

Comment:
Please use substring method to get the file extension and try to use switch case